Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -229,7 +235,11 @@ - (NSDictionary *)liveContext
     });
 
     self.location = !self.location ? [self.configuration shouldUseLocationServices] ? [SEGLocation new] : nil : self.location;
+
+#if TARGET_OS_IOS || (TARGET_OS_MAC && !TARGET_OS_IPHONE)
     [self.location startUpdatingLocation];
+#endif
```

Comment:
I think  we might want to move this lower down - to right before `return [context copy];`?